src/submission-service.ts

```diff
@@ -299,6 +306,24 @@ async function executePostSubmissionAction(
   }
 }
 
+async function uploadAttachment({
+  formId,
+  file,
+}: {
+  formId: number
+  file: UploadAttachmentConfiguration
+}) {
+  const creds = await generateUploadAttachmentCredentials(formId)
+  const result = await uploadAttachmentToS3(creds, file)
+  return {
+    s3: creds.s3,
+    url: result.Location,
```

Comment:
This should be our custom URL e.g.

`https://auth-api-test.blinkm.io/submissions/{formId}/attachments/{attachmentId}`

This is the Url that will go through L@E and authorise downloads.
Last thing I think we want to attach here is the isPrivate flag. May or may not be useful in the future.

Reply:
URL would be 

```js
`${tenants.current.apiOrigin}${creds.s3.key}`
```